src/utils/utils.ts

```diff
@@ -95,3 +95,26 @@ export const hexToRGB = (hex: string, alpha: number) => {
   const rgb = `${r}, ${g}, ${b}`;
   return alpha ? `rgba(${rgb},${alpha})` : `rgb(${rgb})`;
 };
+
+/** Calculate color luminance */
+export const colorLuminance = (r, g, b) => {
```

Comment:
Forgot to ask, are these standard functions for color luminance and contrast ratio for the general web? Just wondering if there should be a note to say that these are functions used elsewhere in other non-nypl projects for luminance and color contrast, and a linked reference?